src/components/KDropdown/KDropdownItem.vue

```diff
@@ -68,6 +68,20 @@ const props = defineProps({
     type: Function,
     default: undefined,
   },
+  /**
+   * @deprecated in favor of `danger`
+   */
+  isDangerous: {
+    type: Boolean,
+    default: false,
+    validator: (value: boolean): boolean => {
+      if (value) {
+        console.warn('KInput: `isDangerous` prop is deprecated. Please use `danger` prop instead. Please see the migration guide for more details: https://alpha--kongponents.netlify.app/guide/migrating-to-version-9.html#kdropdownmenu')
```

Comment:
nit: You're using "please" twice in back-to-back sentences. Can you update the docs for all components to something like this:
```suggestion
        console.warn('KInput: `isDangerous` prop is deprecated. Please use `danger` prop instead. See the migration guide for more details: https://alpha--kongponents.netlify.app/guide/migrating-to-version-9.html#kdropdownmenu')
```

Reply:
Ah sorry, it's prob the Canadian in me 😅
Fixed now